src/frontend/efiling-frontend/src/components/page/home/Home.js

```diff
@@ -16,11 +16,22 @@ export const saveNavigationToSession = ({ cancel, success, error }) => {
   if (error.url) sessionStorage.setItem("errorUrl", error.url);
 };
 
+const addUserInfo = (bceid, firstName, middleName, lastName, email) => {
```

Comment:
change this to:
```
const addUserInfo = (bceID, firstName, middleName, lastName, emailAddress) => {
```
and then below we can just return all key names and no need to specify the value 😸